src/StoreKeeper/WooCommerce/B2C/Exports/OrderExport.php

```diff
@@ -159,14 +160,21 @@ protected function processItem($WpObject)
         /*
          * Billing address
          */
+        $billingHouseNumber = '';
+        if (AddressSearchEndpoint::DEFAULT_COUNTRY_ISO === $WpObject->get_billing_country(self::CONTEXT)) {
+            $houseNumber = $WpObject->get_meta('billing_address_house_number', true);
+            if (!empty($houseNumber)) {
+                $billingHouseNumber = $houseNumber.' ';
+            }
+        }
 
         $callData['billing_address'] = [
             'name' => $WpObject->get_formatted_billing_full_name(),
             'address_billing' => [
                 'state' => $WpObject->get_billing_state(self::CONTEXT),
                 'city' => $WpObject->get_billing_city(self::CONTEXT),
                 'zipcode' => $WpObject->get_billing_postcode(self::CONTEXT),
-                'street' => trim($WpObject->get_billing_address_1(self::CONTEXT)).' '.trim(
+                'street' => $billingHouseNumber.trim($WpObject->get_billing_address_1(self::CONTEXT)).' '.trim(
```

Comment:
you should send house number in `streetnumber` field

Reply:
Properly sending `streenumber` and `flatnumber` now.
![image](https://user-images.githubusercontent.com/18332309/152178702-d03b466e-0dec-4202-b402-f1e18f00bd56.png)